src/fauxtoshop.cpp

```diff
@@ -1,517 +1,487 @@
- * Params: original - image Grid of integers, newImage - image Grid of integers
- * ------------------------------------------------------------
- * Returns: nothing. Void function.
-*/
-
-void compareImages(const GBufferedImage img, GWindow gw){
-    GBufferedImage newImg;
-    //Prompt user to specify a new image file name for comparison
-    nameImageFileToOpen(newImg, gw, false);
-    //Count how many pixels differ between the background and new images
-    int numDiffPixels = img.countDiffPixels(newImg);
-    if (numDiffPixels > 0 ){
-        cout << "These images differ in " << numDiffPixels << " pixel locations!" << endl;
-        // pop up a window to display differences between the two images
-        showDiffWindow(gw, newImg.getFilename());
-    } else {
-        cout << "These images are the same!" << endl;
-    }
-}
-
-/*
- * Function: saveImage
- * Usage: Saves an image after the filter is applied on the image grid.
- * Function:
- * 1. Ask the user whether they'd like to save the image
- * 2. The User enters a file name in order to save the image
- * 3. The user is reprompted if the saving is unsuccesful
- * 4. After saving, clear the screen and print 1 blank line
- * Tip: If the user types "?" as the file name, a graphical file chooser window will pop up
- * to minimise typing for long file names
- * Params: img - GBuggeredImage, imageGrid - Grid<int>
- * ------------------------------------------------------------------------------------------
- * Returns: Nothing. Void function
-*/
-
-void saveImage(GBufferedImage &img, Grid<int> &imageGrid){
-    while (true){
-        //Convert the image grid back into an image for saving
-        img.fromGrid(imageGrid);
-        //Prompt the user to ask if they'd like to save the image
-        string newImageName = getLine("Please enter file name to save image or blank to skip: ");
-        if (newImageName.empty()){
-            //If the user enters blank the image will not be saved
-            cout << "OK, no saving this time. That's cool!" << endl;
-            pause(2000);
-            break;
-        //Save the image if the user enters a valid file name
-        } else if (saveImageToFilename(img, newImageName)){
-            cout << "Saving image as '" << newImageName << "' successful!" << endl;
-            break;
-        }
-        //Reprompt the user if there is an issue with the saving
-        cout << "Oops! Something went wrong... Please try again" << endl;
-    }
-}
+////
+/// This is an implementation of a graphics editing program that allows the user to
+/// edit an image using 4 filters:
+/// a. Scatter - blurs the image
+/// b. Edge detection - creates a black & white image
+/// c. 'Green screen' a sticker image onto a background image
+/// c. Compare an image with another image
+/// After the filter has been applied the user has the option of saving the image
+/// and repeating the filters application on onto the same image or another image
+////
```

Comment:
Header level comments should use the `/* */` syntax. E.g.
```
/*
 * Hello bla bla bla
 * Foo bar baz
 */
```

Reply:
Done!